app/controllers/provider/sessions_controller.rb

```diff
@@ -18,13 +19,15 @@ def new
   def create
     session_return_to
     logout_keeping_session!
-
     @user, strategy = authenticate_user
 
     if @user
       self.current_user = @user
       create_user_session!(strategy.authentication_provider_id)
       flash[:notice] = 'Signed in successfully'
+
+      AuditLogService.call("Signed in: #{self.current_user.username} #{self.current_user.id} #{self.current_user.first_name} #{self.current_user.last_name}")
```

Comment:
I wonder if it makes sense to log both - successful and unsuccessful login attempts.